telegram/ext/filters.py

```diff
@@ -1434,13 +1433,27 @@ def __init__(
         allow_empty: bool = False,
     ):
         super().__init__(chat_id=user_id, username=username, allow_empty=allow_empty)
-        self.chat_id_name = 'user_id'
+        self._chat_id_name = 'user_id'
 
     def get_chat_or_user(self, message: Message) -> Optional[TGUser]:
         return message.from_user
 
     @property
     def user_ids(self) -> FrozenSet[int]:
+        """
+        Which user ID(s) to allow through.
+
+        Warning:
+            :attr:`user_ids` will give a *copy* of the saved user ids as :class:`frozenset`. This
+            is to ensure thread safety. To add/remove a user, you should use :meth:`add_usernames`,
+            :meth:`add_user_ids`, :meth:`remove_usernames` and :meth:`remove_user_ids`. Only update
+            the entire set by ``filter.user_ids/usernames = new_set``, if you are entirely sure
+            that it is not causing race conditions, as this will complete replace the current set
+            of allowed users.
```

Comment:
can you remove the `username` references and add a similar warning to the `usernames` attribute? probably the same for the other _CUBF classes :)

Reply:
I'm going to use the `:inherited-members:` option to solve this, will also help in removing more duplicate code

Reply:
if that works, that'd great 👍 haven't investigated how that clan be applied to only a few classes within `.. automodule` …